CHEATSHEET.md

```diff
@@ -216,32 +221,33 @@ state.memory.store(0x6021f2, aaaa)
 Read Pointer to Pointer from Frame:
 
 ```python
-poi1 = new_state.se.any_int(new_state.regs.rbp)-0x10
-poi1 = new_state.se.any_int(new_state.memory.load(poi1, 8, endness='Iend_LE'))
+poi1 = new_state.solver.eval(new_state.regs.rbp)-0x10
+poi1 = new_state.solver.eval(new_state.memory.load(poi1, 8, endness='Iend_LE'))
```

Comment:
It would be cleaner if this were `new_state.mem[poi1].long.resolved` or `.concrete`

Reply:
also applies to below diffs